test/library/packages/LinearAlgebra/performance/diag/adddiag-test.chpl

```diff
@@ -0,0 +1,61 @@
+/*
+Sparse diagonal creation performance testing
+*/
+
+use LinearAlgebra;
+use LinearAlgebra.Sparse;
+use Time;
+use BlockDist;
+use Math;
+
+config const m=1000000,
+             iters=10,
+             /* Omit timing output */
+             correctness=false;
+
+config type eltType = real;
+
+const bytes = numBytes(eltType);
```

Comment:
This'll no longer compile. `bytes` is now a keyword on master (since #13579). I recommend renaming this var to `nBytes`